helix-term/src/commands/typed.rs

```diff
@@ -1070,6 +1070,9 @@ fn reload_all(
 
         for view_id in view_ids {
             let view = view_mut!(cx.editor, view_id);
+            if !view.doc.eq(&doc_id) {
+                continue;
+            }
 
             view.ensure_cursor_in_view(doc, scrolloff);
```

Comment:
I think it's a little clearer to check for the positive case:

```rust
if view.doc == doc_id {
  view.ensure_cursor_in_view(doc, scrolloff);
}
```

Reply:
I usually prefer returning early and perform any actions afterwards but I don't mind using your suggestion instead.

Reply:
done